packages/editor-ui/src/Interface.ts

```diff
@@ -856,8 +856,19 @@ export interface INodeMetadata {
 	parametersLastUpdatedAt?: number;
 }
 
-export interface IRootState {
+export interface IExecutionsState {
 	activeExecutions: IExecutionsCurrentSummaryExtended[];
```

Comment:
can we rewrite this as a mapping from id to execution object... I was expecting something like this in the store
```suggestion
	activeExecutions: {} as {[id: string]: IExecutionsCurrentSummaryExtended[];
	activeExecutionId: null as null | number;
```
where the activeExecutionId is the currently open execution when you are an open execution page..